data/tlg0007/tlg072/__CTS__.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0007"
+    projid="greekLit:tlg072" urn="urn:cts:greekLit:tlg0007.tlg072" xml:lang="grc">
+    <ti:title xml:lang="lat">De capienda ex inimicis utilitate</ti:title>
+
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg072.perseus-eng3"
+        workUrn="urn:cts:greekLit:tlg0007.tlg072" xml:lang="eng">
+        <ti:label xml:lang="eng">How To Profit From One's Enemies</ti:label>
+        <ti:description xml:lang="mul">Plutarch. Moralia, Vol. II. Babbitt, Frank Cole, translator.
+            Cambridge, MA: Harvard University Press. London: William Heinemann Ltd. 1927
+            (printing).</ti:description>
+    </ti:translation>
+
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg072.perseus-eng4"
+        workUrn="urn:cts:greekLit:tlg0007.tlg072" xml:lang="eng">
+        <ti:label xml:lang="eng">How a man may receive advantage and profit from his
+            enemies.</ti:label>
+        <ti:description xml:lang="eng">Plutarch. Plutarch's Morals, Vol. II. Goodwin, William W.,
```

Comment:
This is Vol.  I of Goodwin. Be careful when copying other cts work files. The volumes and dates are going to be all over the place.

Reply:
The Perseus Catalog will the have the best info on volumes and dates if the file itself cannot be found but always check the book if you can find it.